main.js

```diff
@@ -0,0 +1,201 @@
+
+
+
+$('document').ready(function() {
+
+  //Populate page with user set default location if set
+  if(localStorage.getItem('isDefault')) {
+    let weatherData = JSON.parse(localStorage.getItem('data'))
+    $('#set-location-container').fadeIn('slow')
+    showCurrentWeather(weatherData);
+    showFiveDayWeather(weatherData);
+  }
+  
+
+  //ANCHOR Function to call Weather API to get Current and Five Day weather
+  const getWeatherData = function(){
+    const cityValue = $('#user-location').val();
+    const stateValue = $('#user-state').val()
+
+    //API call to get the Lat and Lon of searched city/state
+    async function getLocationWeather() {
+      try {
+        const latLonResponse = await fetch(`http://api.openweathermap.org/geo/1.0/direct?q=${cityValue},${stateValue}&limit=1&appid=${keys.weatherKey}`)
```

Comment:
It's actually possible to search for weather/forecast by city name, which would make the lat/long unnecessary.